addons/wowl/static/src/legacy/legacy.ts

```diff
@@ -98,13 +98,42 @@ odoo.define("wowl.ActionAdapters", function (require: any) {
 
   class ActionAdapter extends ComponentAdapter {
     am = useService("action_manager");
+    router = useService("router");
+
+    constructor(...args: any[]) {
+      super(...args);
+      const _trigger_up = this._trigger_up.bind(this);
+      let pushedState: any;
+      this._trigger_up = (ev: any) => {
+        if (ev.name === "push_state") {
+          pushedState = Object.assign(pushedState || {}, ev.data.state);
+          return;
+        }
+        return _trigger_up(ev);
+      };
+      hooks.onMounted(() => {
+        if (pushedState) {
+          // push the state after the actionManager to avoid being wiped out by it
+          setTimeout(() => {
```

Comment:
this will be a problem... we don't want people to have to think of this when then want to push something in the url